Jenkinsfile-smoke.groovy

```
@@ -20,5 +20,6 @@ if (comment) {
         ocDeployerComponentPath: "aiops/aiops-data-collector",
```

@anurag03 I think this is the issue. We're in the `aiops-incoming-listener` repo, but Jenkins tries to build the data collector instead.
```suggestion
        ocDeployerComponentPath: "aiops/aiops-incoming-listener",
```